base/shell/explorer/lang/pt-PT.rc

```diff
@@ -187,7 +187,7 @@ BEGIN
     GROUPBOX         "Seleccione o tamanho do icone dos  programas ", IDC_STATIC, 7, 7, 238, 37
     ICON             IDC_LARGEICON, 17, 18, 20, 20, SS_CENTERIMAGE
     ICON             IDC_SMALLICON, 151, 18, 20, 20, SS_CENTERIMAGE
-    AUTORADIOBUTTON  "Icons &grandes", IDC_CHOOSELARGE, 43, 25, 53, 10
+    AUTORADIOBUTTON  "Icones &grandes", IDC_CHOOSELARGE, 43, 25, 53, 10
     AUTORADIOBUTTON  "icones &pequenos", IDC_CHOOSESMALL, 177, 25, 51, 10, NOT WS_TABSTOP
```

Comment:
```suggestion
    AUTORADIOBUTTON  "Icones &pequenos", IDC_CHOOSESMALL, 177, 25, 51, 10, NOT WS_TABSTOP
```

Reply:
@zecarlos1957 Could you check whether pt-BR also uses lowercase 'i' here as well? If so I suppose it may also need a change.

Reply:
This file has not yet been translated